cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -115,8 +115,8 @@ class AnalyzerCommand : OrtCommand(
 
     private val repositoryConfigurationFile by option(
         "--repository-configuration-file",
-        help = "A file containing the repository configuration. If set, the '$ORT_REPO_CONFIG_FILENAME' file from " +
-                "the repository is ignored."
+        help = "A file containing the repository configuration. If set, overrides any repository configuration " +
```

Comment:
Would it make sense to simplify like follows? (up to you)

The filepath to the repository configuration file, relative to the repository root. Defaults to '$ORT_REPO_CONFIG_FILENAME'.